publish-extensions.js

```diff
@@ -135,8 +136,10 @@ const flags = [
       }
 
       let timeout;
-      await new Promise((resolve, reject) => {
-        const p = cp.spawn(process.execPath, ['publish-extension.js', JSON.stringify(extension)], {
+      await new Promise(async (resolve, reject) => {
+        const repository = extension.download && extension.download.replace(/\/releases\/download\/.*$/, '');
```

Comment:
Let's fail if msVersion is defined, but extension.repository is undefined. It means that we should manually add repository in extensions.json.